pkg/metrics/register.go

```diff
@@ -24,7 +25,7 @@ var (
 )
 
 func (t *taskMetrics) RegisterMetric(name string, help string, labels []string, handler MetricHandler) error {
-	labels = append(labels, appNameLabel, appVersionLabel)
+	labels = append(labels, appNameLabel, appVersionLabel, appRevisionLabel)
```

Comment:
appName can be a label, but appVersion and appRevison, as well as appStartUp should be a metric itself